src/main/java/com/github/tomakehurst/wiremock/matching/MatchesXPathPattern.java

```diff
@@ -59,10 +54,8 @@ public MatchesXPathPattern(
 
   public MatchesXPathPattern withXPathNamespace(String name, String namespaceUri) {
     Map<String, String> namespaceMap =
-        ImmutableMap.<String, String>builder()
-            .putAll(getFirstNonNull(xpathNamespaces, Collections.emptyMap()))
-            .put(name, namespaceUri)
-            .build();
+        new HashMap<>(getFirstNonNull(xpathNamespaces, Collections.emptyMap()));
```

Comment:
Should it be kept immutable?

Reply:
Hm...
All works fine

```
        Map<String, String> map1 = new HashMap<>(Collections.emptyMap());
        map1.put("1", "1");
        System.out.println(map1);

        Map<String, String> map2 = new HashMap<>(new HashMap<>());
        map2.put("2", "2");
        System.out.println(map2);
```

Rollback to Collections.emptyMap()?

Reply:
Thanks for adding a wrapper. Just in case